examples/Training/java/src/training/HowToUseTables.java

```diff
@@ -114,6 +123,25 @@ private void createTable()
             table.addData(newRow);
             OriginalFile file = table.getOriginalFile(); // if you need to interact with the table
             file = new OriginalFileI(file.getId(), false);
+
+            DataManagerFacility dm = gateway.getFacility(DataManagerFacility.class);
+
+            FileAnnotation annotation = new FileAnnotationI();
+            annotation.setFile(file);
+            annotation.setNs(omero.rtypes
+                    .rstring(omero.constants.namespaces.NSBULKANNOTATIONS.value));
+            
+            //The following saveAndReturnObject call can be removed in bulk annotation scenarios
+            //The server call is made when a link is created and that handles both the annotation and the link together
+            annotation = (FileAnnotation) dm.saveAndReturnObject(ctx, annotation);
+
+            //now link the image and the annotation
+            ImageAnnotationLink link = new ImageAnnotationLinkI();
+            link.setChild(annotation);
+            link.setParent(image.asImage());
```

Comment:
@bramalingam: you need to add again the ``loadImage`` method
what was necessary was to change the constructor so the image ID is the image ID read from the configuration file or the hard coded one if you run using the main method in the class.
i.e.
you need to add 
``
private void loadImage(long imageId) throws Exception {
        BrowseFacility browse = gateway.getFacility(BrowseFacility.class);
        ImageData image = browse.getImage(ctx, imageId);
}
``
and in the constructor invoked the method